src/app/components/PercentageValue/index.tsx

```diff
@@ -7,13 +7,17 @@ type PercentageValueProps = {
   total?: number | string
   value: number | string | undefined
   maximumFractionDigits?: number
+  threshold?: number
+  fractionDigitsForSmallValues?: number
 }
 
 export const PercentageValue: FC<PercentageValueProps> = ({
   maximumFractionDigits,
   adaptMaximumFractionDigits = 2,
```

Comment:
Was this supposed to have been:
```suggestion
  maximumFractionDigits = 2,
  adaptMaximumFractionDigits = true,
```